Source/ViewModel/MLCardFormViewModel.swift

```diff
@@ -321,7 +321,7 @@ final class MLCardFormViewModel {
         guard let fieldId = MLCardFormFields(rawValue: cardFormField.property.fieldId()) else { return (false, nil) }
         if fieldId == MLCardFormFields.cardNumber,
             cardFormField.property.isValid(value: cardFormField.getValue()),
-            let currentBin = cardFormField.getValue()?.replacingOccurrences(of: " ", with: "").prefix(6),
+            let currentBin = cardFormField.getValue()?.replacingOccurrences(of: " ", with: "").prefix(8),
```

Comment:
the change looks good. but....... what do we really intend to do here. 
is posible to handle both 6 and 8 digits? maybe using contains?

Reply:
We validated it with backend and we need to validate with 8 digits from now on